include/alpaka/KernelBundle.hpp

```diff
@@ -28,23 +28,35 @@ namespace alpaka
         using ArgTuple = std::tuple<remove_restrict_t<std::decay_t<TArgs>>...>;
 
         // Constructor
-        constexpr KernelBundle(KernelFn kernelFn, TArgs&&... args)
-            : m_kernelFn{std::move(kernelFn)}
+        constexpr KernelBundle(KernelFn const& kernelFn, TArgs&&... args)
```

Comment:
```suggestion
        constexpr KernelBundle(KernelFn const& kernelFn, auto&&... args)
```

Otherwise `args` are rvalue references.

Reply:
I will remove `&&` with `const&`